### DISCUSSION TITLE
Github pages for commercial landing page

### DISCUSSION THREAD

**Discussion by nitroboy720** [community/community#33114] [Other Feature Feedback, Questions, & Ideas]
Hi, is it okay with Github to use Github pages to host a landing page for a SASS (landing page, documentation, demo request and FAQ) ?

I saw this: 

`Prohibited uses
GitHub Pages is not intended for or allowed to be used as a free web-hosting service to run your online business, e-commerce site, or any other website that is primarily directed at either facilitating commercial transactions or providing commercial software as a service (SaaS). GitHub Pages sites shouldn't be used for sensitive transactions like sending passwords or credit card numbers.`

but it is not clear to me if this applies to a non transactional site. Just the landing page, the FAQs, and documentation etc.
Also, it says it is not to be used as a free web-hosting service. But i do not mind upgrading to Pro or paying for the service, I just prefer it versus other hosting options.

Any way to use Github pages for this purpose?

Thanks

**Comment by radar-blip** [comment]
I think a landing page would be okay, but I'm not 100% sure. You may want to wait for others to chime in.

**Comment by jfullstackdev** [comment]
for a full commercial one, I don't think so

but say, you just want to have a little bit of donations from others from your open source projects / tutorials, that's allowed
